src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int side;
+    private final int upSide;
+    private final int downSide;
```

Comment:
Use meaningful names for variables.
```suggestion
    private final int leg;
    private final int topBase;
    private final int bottomBase;
```